### DISCUSSION TITLE
Error occurred: Failed to fetch

### DISCUSSION THREAD

**Discussion by Maffeii** [janhq/jan#1539] [Get Help] [answered]
When I'm trying to use Jan.
I got this error;
Error occurred: Failed to fetch

The module I'm trying to use:
Mixtral 8x7B Instruct Q4

My system:
Apple M1 Pro
32 GB RAM
 
Edit:
If it's an allocation problem (insufficient RAM), it would be nice to see it that way. Error message isn't very helpful at all.

**Comment by hahuyhoang411** [comment]
You can try smaller model since Mixtral 7B need 40-ish GB of RAM. With 32GB you can try 34B models.

**Comment by sjmcdonald6** [comment]
I've encountered the same issue. 
"Error occurred: Failed to fetch"

But I've also encountered this message:
"Error occurred: network error"

The model I'm using is also:
Mixtral 8x7B Instruct Q4

My system is:
Apple M1 Max
64 GB RAM

I've tried reloading, forced reload, quit the app, and restarted computer,
But nothing seems to work. 
My CPU usage has not gone above 20%, and my Memory usage has not gone above 50%.

I will attempt to switch to a smaller model to see if that helps.

**Comment by louis-jan** [comment, marked as answer]
The app now handles errors better and will provide a clear error message when there's an issue with loading the model. It appears that these models are outdated, and you'll need to find a newer quantized version of them.
